external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MsSqlServerIntegrationSuite.scala

```diff
@@ -26,9 +26,10 @@ import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.tags.DockerTest
 
 /**
- * To run this test suite for a specific version (e.g., 2019-CU13-ubuntu-20.04):
+ * To run this test suite for a specific version (e.g., 2019-CU14-ubuntu-20.04):
  * {{{
- *   ENABLE_DOCKER_INTEGRATION_TESTS=1 MSSQLSERVER_DOCKER_IMAGE_NAME=2019-CU13-ubuntu-20.04
+ *   ENABLE_DOCKER_INTEGRATION_TESTS=1
+ *   MSSQLSERVER_DOCKER_IMAGE_NAME=mcr.microsoft.com/mssql/server:2019-CU14-ubuntu-20.04
```

Comment:
As far as I could tell the referenced image needs to be pulled from mcr.microsoft.com and the previously referenced image doesn't seem to exist anymore. https://hub.docker.com/_/microsoft-mssql-server

Reply:
Also, this example was missing the repository name, so I added that

Reply:
cc @sarutak FYI

Reply:
@tdg5 Thank you for fixing the example. Could you fix the example in `v2/MsSqlServerIntegrationSite` too?

Reply:
Also, please change [this part](https://github.com/apache/spark/pull/34745/files#diff-cb12fa58307c2155fe2156d347d7a7917f1c755c34488a741dfb06653f83f364L40) too.

Reply:
hmm, this image also seems to exist. I'm going to roll that part back, but will fix the docs in another PR.